programming/autonomous.md

```diff
@@ -6,13 +8,15 @@ To create a state for autonomous, you have to import the AutonomousStateMachine.
 You also need to import any components that you will be using from the components subdirectory.
 
 ```
+python
```

Comment:
I believe we need to have the language, `python` be on the same line as the three backticks ... as in:

    ```python
    from magicbot import ...